docs/Installation.md

```diff
@@ -514,14 +518,22 @@ psql -U pgsql -f ./initial-postgres.sql template1
 
 ### 3.3 Service startup (FreeBSD)
 
->
-> At this time there is no instruction for automatically running Zonemaster Web backends
-> nor Workers as services on FreeBSD.
->
+Install service scripts:
+
+```sh
+install -m 775 -g zonemaster -d /var/log/zonemaster
+install -m 775 -g zonemaster -d /var/run/zonemaster
+install -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
+install -m 755 ./zm_testagent-bsd /usr/local/etc/rc.d/zm_testagent
+echo 'zm_rpcapi_enable="YES"' | tee -a /etc/rc.conf
+echo 'zm_testagent_enable="YES"' | tee -a /etc/rc.conf

```

Comment:
I propose the following instead:

sysrc zm_rpcapi_enable="YES"
sysrc zm_testagent_enable="YES"

Reply:
Sorry, I forgot that.. Addressed it now.